src/EditorFeatures/Core.Cocoa/Lsp/VSMacWorkspaceRegistrationService.cs

```diff
@@ -13,16 +13,11 @@ namespace Microsoft.CodeAnalysis.EditorFeatures.Cocoa.Lsp;
 /// Implementation of the workspace registration service exported to be included in the VSMac composition.
 /// </summary>
 [Export(typeof(LspWorkspaceRegistrationService)), Shared]
-internal class VSMacWorkspaceRegistrationService : LspWorkspaceRegistrationService
+internal sealed class VSMacWorkspaceRegistrationService : LspWorkspaceRegistrationService
```

Comment:
Are you potentially doing a follow-up to remove the derived types, if they're all the same now?

Reply:
Potentially. But it's low on my list.